src/libs/ajax/workflows-app/CromwellApp.js

```diff
@@ -22,4 +22,19 @@ export const CromwellApp = (signal) => ({
       },
     };
   },
+  callCacheDiff: async (cromwellUrlRoot, thisWorkflow, thatWorkflow) => {
```

Comment:
That's definitely valid... there's a larger refactor that would be nice to do, to collapse the CromIAM and CromwellApp ajax interfaces into a single thing that can figure out what Cromwell to call based on which workspace is relevant. I avoided that because I didn't want to change everything that used either set of endpoints. And I struggled to find something nicer than this that didn't require that.